.github/workflows/test.yml

```diff
@@ -21,8 +21,8 @@ jobs:
       - run: npm ci # throws an error if the package-lock.json file is out of sync
       - name: Check for linting errors (run `npm run lint:fix` to fix)
         run: npm run lint
-      - name: Check for prettier errors (run `npm run format` to fix)
-        run: npx prettier --check .
+      # - name: Check for prettier errors (run `npm run format` to fix)
+      #   run: npx prettier --check .
```

Comment:
By the way, you should be able to re-enable this after you run `npm run format` and commit the changes.

(It's basically just adding a trailing comma to everything: https://prettier.io/blog/2023/07/05/3.0.0.html#javascript)